wsdl/ver10/schema/onvif.xsd

```diff
@@ -6728,6 +6728,12 @@ If set to 0.0, infinity will be used.</xs:documentation>
 			<xs:enumeration value="Any"/>
 		</xs:restriction>
 	</xs:simpleType>
+	<xs:simpleType name="ObjectMovement">
```

Comment:
We already have a tt:Direction type. Should it be extended if you want to add "depth" to the direction?

Reply:
we analysed to use tt:Direction but it is built on using 'polylines' which does not fit very well when using area/zone for validating approaching/leaving.

Reply:
I read both your and @jflevesque-genetec comments and the section on LPR and echo his same question, tt:Direction type. Should it be extended? and change the use of 'polylines"?